packages/node_modules/@webex/internal-plugin-conversation/src/share-activity.js

```diff
@@ -8,6 +8,7 @@ import {proxyEvents, transferEvents} from '@webex/common';
 import {WebexPlugin} from '@webex/webex-core';
 import {filter, map, pick, some} from 'lodash';
 import {detectFileType, processImage} from '@webex/helper-image';
+import sha256 from 'crypto-js/sha56';
```

Comment:
Unit tests are failing due to a typo here. 

Should be `from 'crypto-js/sha256';`

Reply:
Oops. 😬 

Updated it, please check.